### PR TITLE
Hyrax 3.0 map

### PR DESCRIPTION
**Pull request by jlhardes**
Adds a page to the Metadata Application Profile section to update the MAP for Hyrax 3.0.

### CODE REVIEW

**Review by elrayle — APPROVED** [review]
I reviewed that the pages display as expected.  I minimally reviewed content as metadata is not my area of expertise.

File: pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+| keyword          | schema:keywords | ::RDF::Vocab::SCHEMA.keywords | MAY | xsd:string (Literal) |
+| license          | dct:license | ::RDF::Vocab::DC.license | MAY | xsd:anyURI |
+| _access_right_     | _dct:accessRights_ | _::RDF::Vocab::DC.accessRights_ | _MAY_ | _xsd:string (Literal)_ |
+| _rights_notes_     | _dct:rights_ | _::RDF::Vocab::DC.rights_ | _MAY_ | _xsd:string (Literal)_ |
+| publisher        | dce:publisher | ::RDF::Vocab::DC11.publisher | MAY | xsd:string (Literal) |
+| date_created     | dct:created | ::RDF::Vocab::DC.created | MAY | xsd:date or xsd:dateTime xsd:string (Literal) |
+| subject          | dce:subject | ::RDF::Vocab::DC11.subject | MAY | xsd:string (Literal) |
+| language         | dce:language | ::RDF::Vocab::DC11.language | MAY | xsd:string (Literal) |
+| identifier       | dct:identifier | ::RDF::Vocab::DC.identifier | MAY | xsd:string (Literal) |
+| based_near       | foaf:basedNear | ::RDF::Vocab::FOAF.based_near | MAY | xsd:anyURI |
```
Comment (wickr) [inline pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md:72]: Predicate needs to have `foaf:based_near` instead. Rdf-vocab part is correct.

File: pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+| creator          | dce:creator | ::RDF::Vocab::DC11.creator | MUST (Required) | xsd:string (Literal) |
+| rights_statement | edm:rights | ::RDF::Vocab::EDM.rights | MUST (Required) | xsd:anyUri |
+| alternative_title | dct:alternative | ::RDF::Vocab::DC.alternative | MAY | xsd:string (Literal) |
+| contributor      | dce:contributor | ::RDF::Vocab::DC11.contributor | MAY | xsd:string (Literal) |
+| description      | dce:description | ::RDF::Vocab::DC11.description | MAY | xsd:string (Literal) |
+| _abstract_         | _dct:abstract_ | _::RDF::Vocab::DC.abstract_ | _MAY_ | _xsd:string (Literal)_ |
+| keyword          | schema:keywords | ::RDF::Vocab::SCHEMA.keywords | MAY | xsd:string (Literal) |
+| license          | dct:license | ::RDF::Vocab::DC.license | MAY | xsd:anyURI |
+| _access_right_     | _dct:accessRights_ | _::RDF::Vocab::DC.accessRights_ | _MAY_ | _xsd:string (Literal)_ |
+| _rights_notes_     | _dct:rights_ | _::RDF::Vocab::DC.rights_ | _MAY_ | _xsd:string (Literal)_ |
```
Comment (wickr) [inline pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md:66]: The DC Elements Rights is used in the definition, so these should be `dce:rights` and `::RDF::Vocab::DC11.rights`

**Review by wickr — CHANGES_REQUESTED** [review]
A few changes needed.

File: pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+| publisher        | dce:publisher | ::RDF::Vocab::DC11.publisher | MAY | xsd:string (Literal) |
+| date_created     | dct:created | ::RDF::Vocab::DC.created | MAY | xsd:date or xsd:dateTime xsd:string (Literal) |
+| subject          | dce:subject | ::RDF::Vocab::DC11.subject | MAY | xsd:string (Literal) |
+| language         | dce:language | ::RDF::Vocab::DC11.language | MAY | xsd:string (Literal) |
+| identifier       | dct:identifier | ::RDF::Vocab::DC.identifier | MAY | xsd:string (Literal) |
+| based_near       | foaf:basedNear | ::RDF::Vocab::FOAF.based_near | MAY | xsd:anyURI |
+| related_url      | rdfs:seeAlso | ::RDF::RDFS.seeAlso | MAY | xsd:string or xsd:anyURI |
+| source           | dct:source | ::RDF::Vocab::DC.source | MAY | xsd:string (Literal) |
+| resource_type    | dct:type | ::RDF::Vocab::DC.type | MAY | xsd:string (Literal) |
+| bibliographic_citation* | dct:bibliographicCitation | ::RDF::Vocab::DC.biliographic_citation | MAY | xsd:string (Literal) |
```
[the remark this answers is not in the document]
Comment (jlhardes) [reply pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md]: Thanks, made this change!

File: pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md (hunk 1 above)
Comment (jlhardes) [reply pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md]: Thanks, made this change too!

File: pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md (hunk 2 above)
Comment (jlhardes) [reply pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md]: And made this change!

File: pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (wickr) [reply pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md]: Ack, one more, I didn't see this earlier, but there's a `b` missing from bibliographic

[the remark this answers is not in the document]
Comment (jlhardes) [reply pages/samvera/developer_resources/customize_metadata/hyrax_3.0/metadata_application_profile.md]: Thank you for looking through all of these! I made this change as well. Hopefully we have all of the fields in good order now.

**Review by wickr — APPROVED** [review]

**Pull request merged**